applications/tari_app_grpc/proto/wallet.proto

```diff
@@ -57,6 +62,29 @@ message TransferResult {
     string failure_message = 4;
 }
 
+message GetTransactionInfoRequest {
+    repeated uint64 transaction_ids = 1;
+}
+
+message GetTransactionInfoResponse {
+    repeated TransactionInfo statuses = 1;
+}
+
+message TransactionInfo {
+    uint64 tx_id = 1;
+    string address = 2;
```

Comment:
Not `bytes` maybe?

Reply:
A string was chosen because it makes storage easier (blob vs varchar, select query) and removes the need for a hex / base 58 dependency and conversions. In effect, the caller needs to know how to tari converts the byte (i.e bincode) representation to a string. If this field is a "canonical string representation of an address", we can treat this string a like a unique id without caring what it encodes.

Reply:
I don't know. GRPC handles byte arrays really well and all of the other protos are defined as bytes. I would say we should keep it consistent as bytes, or change all places to use a canonical string representation. This change would also be needed to be applied to `tari_crypto`

Reply:
Ok fair enough, I'll make the change